week3/src/components/CardContainer.jsx

```diff
@@ -0,0 +1,145 @@
+import { useMemo, useState, useEffect } from "react"
+import styled from 'styled-components';
+import CARDS from '../constants/card';
+import Card from "./Card";
+import ClearModal from "./ClearModal";
+import { LEVEL } from "../constants/Level";
+
+export default function CardContainer({updateScore}) {
+
+    // // 게임이 끝났는지 저장
+    const [finished, setFinished] = useState(false)
+    // 현재 선택한 카드 데이터 저장하는 배열
+    const [clicked, setClicked] = useState(Array.from([]))
```

Comment:
Array.from은 Set과 같은 객체를 배열로 변환해주는 메서드에요!! 지금 Array.from의 인자로 빈 배열이 들어갔는데 빈 배열 자체가 배열이므로 Array.from([])은 []과 같아요!! 그래서 가독성을 위해 이렇게 작성해주는 게 좋겠네요!!
```suggestion
    const [clicked, setClicked] = useState([])
```


또 Array.from은 생성 배열의 각 요소에 대해 함수를 적용할 수 있는데, 예시를 하나 보여드릴게욥!!

```js
const numArr = Array.from({ length: 5 }, (_, i) => i);
console.log(numArr); // [0, 1, 2, 3, 4]
```